src/pages/account-menu/in-an-app-bar/index.tsx

```diff
@@ -0,0 +1,348 @@
+    },
+    textContainer: {
+        marginLeft: `${theme.spacing(2.5)}px`,
+        overflow: 'hidden',
+        whiteSpace: 'nowrap',
+    },
+    paper: {
+        marginTop: `${theme.spacing(1)}px`,
+    },
+    userMenuChip: {
+        width: '112px',
+        height: `${theme.spacing(4)}px`,
+    },
+    chipIcon: {
+        height: `${theme.spacing(3)}px`,
+        width: `${theme.spacing(3)}px`,
+    },
+}));
+
+export const InAnAppBar = (): JSX.Element => {
+    const dispatch = useDispatch();
+    const classes = useStyles();
+    const [chipToggled, setChipToggled] = useState(false);
+
+    const toggleChip = useCallback((): void => {
+        setChipToggled(!chipToggled);
+    }, [chipToggled]);
+
+    return (
+        <div style={{ minHeight: '100vh' }}>
+            <AppBar data-cy="pxb-toolbar" position={'sticky'}>
+                <Toolbar classes={{ gutters: classes.toolbarGutters }}>
+                    <Hidden mdUp={true}>
+                        <IconButton
+                            data-cy="toolbar-menu"
+                            color={'inherit'}
+                            onClick={(): void => {
+                                dispatch({ type: TOGGLE_DRAWER, payload: true });
+                            }}
+                            edge={'start'}
+                            style={{ marginRight: 20 }}
+                        >
+                            <MenuIcon />
+                        </IconButton>
+                    </Hidden>
+                    <Typography variant={'h6'} color={'inherit'}>
+                        In an App Bar
+                    </Typography>
+                    <div />
+                </Toolbar>
+            </AppBar>
+            <div>
+                <div className={classes.appBarHeader}>
+                    <Typography variant={'body1'}>
+                        Click on each avatar to see the account menu. Resize the screen to view the account menu / user
+                        menu rendered responsively.
+                    </Typography>
+                </div>
+                <div className={classes.appBarContainer}>
+                    {/* Generic Icon Avatar Example */}
+                    <AppBar position="static" color="inherit" className={classes.appBar}>
+                        <Toolbar classes={{ gutters: classes.toolbarGutters }}>
+                            <IconButton edge={'start'}>
+                                <MenuIcon />
+                            </IconButton>
+                            <div className={classes.textContainer}>
+                                <Typography variant={'h6'} noWrap>
+                                    Generic Icon Avatar
+                                </Typography>
+                                <Typography variant={'body1'} noWrap>
+                                    Shared / Anonymous Account
+                                </Typography>
```

Comment:
the title/subtitle styles on these AppBars doesn't look like they match up with our specs (spacing is off maybe the size/weight too). Does our ThreeLiner component give us the right default styles if we were to use that here?

Reply:
If I use Threeliner component. 
![image](https://user-images.githubusercontent.com/10433274/127197736-4e2c8a86-c3dc-4513-8d7e-0a0165b37701.png) need to update font-size and font-weight to match designs.

In current implementation using typography font-size and font-weights are as per design. As per https://github.com/pxblue/react-design-patterns/pull/67#issuecomment-886759103 if add `margin-top: -4px;` it looks like below image.
![image](https://user-images.githubusercontent.com/10433274/127198073-2601ac73-215c-4950-b964-afa550381666.png)

Reply:
Your last screenshot looks correct to me with the negative margin.